ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/LineaProtocolSpecs.java

```diff
@@ -62,6 +77,10 @@ static ProtocolSpecBuilder lineaDefinition(
                         TransactionType.EIP1559),
                     txCalldataMaxSize))
         .blockBodyValidatorBuilder(LineaBlockBodyValidator::new)
+        // some Linea evm opcodes behave differently.
+        .evmBuilder(
+            (gasCalculator, jdCacheConfig) ->
+                MainnetEVMs.linea(gasCalculator, chainId.orElse(BigInteger.ZERO), evmConfiguration))
```

Comment:
Create a new definition, for example `lineaOpCodesChangesDefinition`, that is based on `lineaDefinition` and has these EVM changes, since they should not be enabled yet

Reply:
Sure! will do

Reply:
done